flocker/node/agents/test/test_blockdevice_manager.py

```diff
@@ -0,0 +1,138 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.node.agents.blockdevice_manager``.
+"""
+
+from uuid import uuid4
+
+from twisted.trial.unittest import SynchronousTestCase
+from zope.interface.verify import verifyObject
+
+from ..blockdevice_manager import (
+    IBlockDeviceManager,
+    BlockDeviceManager,
+    MountInfo,
+    MakeFilesystemError,
+    MountError,
+    UnmountError,
+)
+
+from .test_blockdevice import (
+    loopbackblockdeviceapi_for_test,
+    LOOPBACK_MINIMUM_ALLOCATABLE_SIZE,
+    mountroot_for_test,
+)
+
+
+class BlockDeviceManagerTests(SynchronousTestCase):
+    """Tests for flocker.node.agents.blockdevice_manager.BlockDeviceManager."""
+
+    def setUp(self):
+        """Establishes testing infrastructure for test cases."""
```

Comment:
Couple of nits:

 1. Docstrings are generally in the imperative, e.g. "Establish testing infrastructure..."
 2. Triple quotes go on their own lines.